backend/src/main/java/botobo/core/exception/GlobalControllerAdvice.java

```diff
@@ -1,97 +1,71 @@
 import org.springframework.web.bind.MethodArgumentNotValidException;
 import org.springframework.web.bind.MissingServletRequestParameterException;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
 import org.springframework.web.multipart.MaxUploadSizeExceededException;
 
-import java.util.stream.Collectors;
+import java.util.Objects;
 
 @RestControllerAdvice
 @Slf4j
 public class GlobalControllerAdvice {
 
-    @ExceptionHandler(NotFoundException.class)
-    public ResponseEntity<ErrorResponse> handleNotFoundException(NotFoundException e) {
-        log.info("NotFoundException", e);
-        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(ErrorResponse.of(e.getMessage()));
-    }
-
-    @ExceptionHandler(UnAuthorizedException.class)
-    public ResponseEntity<ErrorResponse> handleUnauthorizedException(UnAuthorizedException e) {
-        log.info("UnauthorizedException", e);
-        return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(ErrorResponse.of(e.getMessage()));
-    }
-
     @ExceptionHandler(MethodArgumentNotValidException.class)
     public ResponseEntity<ErrorResponse> handleBindingException(MethodArgumentNotValidException e) {
-        String message = e.getFieldErrors()
-                .stream()
-                .map(DefaultMessageSourceResolvable::getDefaultMessage)
-                .collect(Collectors.joining(System.lineSeparator()));
-        log.info("MethodArgumentNotValidException", e);
-        return ResponseEntity.badRequest().body(ErrorResponse.of(message));
-    }
+        FieldError fieldError = e.getFieldError();
+        Objects.requireNonNull(fieldError);
 
-    @ExceptionHandler(MissingServletRequestParameterException.class)
-    public ResponseEntity<ErrorResponse> handleMissingParams(MissingServletRequestParameterException e) {
-        String message = String.format("파라미터를 입력해야 합니다.(%s)", e.getParameterName());
+        ErrorType errorType = ErrorType.of(fieldError.getDefaultMessage());
         log.info("MethodArgumentNotValidException", e);
-        return ResponseEntity.badRequest().body(ErrorResponse.of(message));
+        return ResponseEntity.badRequest().body(ErrorResponse.of(errorType));
     }
 
-    @ExceptionHandler(BadRequestException.class)
-    public ResponseEntity<ErrorResponse> handleBadRequestException(BadRequestException e) {
-        log.info("BadRequestException", e);
-        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(ErrorResponse.of(e.getMessage()));
+    @ExceptionHandler(InternalServerErrorException.class)
+    public ResponseEntity<ErrorResponse> handleInternalServerErrorException(InternalServerErrorException e) {
+        log.info("InternalServerErrorException", e.getServerMessage());
+        ErrorType errorType = ErrorType.of(e.getClass());
+        return ResponseEntity.status(e.getHttpStatus()).body(ErrorResponse.of(errorType));
```

Comment:
```suggestion
        return ResponseEntity.status(e.getHttpStatus()).body(ErrorResponse.of(e.getErrorType()));
```
다른 곳처럼 여기도 이렇게 바꿔보는건 어떨까요?